src/controllers/requestController.js

```diff
@@ -18,23 +18,53 @@ const requestTrip = async (req, res) => {
     const { body, decoded } = req;
     const { id: userId } = decoded;
     const {
-      type, originCity, destinationCity, departureDate, returnDate, reason, accommodation
+      type, originCity, destinationCity, departureDate, returnDate, reason,
```

Comment:
is subRequest validated on the request trip schema?

Reply:
yes, subRequest is validate on the request trip schema now.